config.json.sample

```diff
@@ -18,6 +18,18 @@
     	},
         
     	"S3" : {
-        }
+        }, 
+
+        "OneClick" : {
+            "username" : "app-agent-username",
+            "password" : "app-agent-pwd",
+            "library_id" : "participating-library-id",
+            "remote_stage" : "qa", 
+            "url" : "https://api.oneclickdigital.us/", 
+            "basic_token" : "abc-def-ghi", 
+            "ebook_loan_length" : "integer-number-days", 
+            "eaudio_loan_length" : "integer-number-days"
+        },
+
```

Comment:
Rather than adding this to the sample config, you should use `temp_config()` to create a new Configuration object during any test that needs to have OneClick configured. 

See tests/test_novelist.py for a relevant example.